diff --git a/modules/media/src/MediaStorage.php b/modules/media/src/MediaStorage.php
index c66476a..3beb51d 100644
--- a/modules/media/src/MediaStorage.php
+++ b/modules/media/src/MediaStorage.php
@@ -17,6 +17,12 @@ class MediaStorage extends SqlContentEntityStorage {
    * {@inheritdoc}
    */
   public function save(EntityInterface $media) {
+    /** @var \Drupal\ls_migration\MigrationStatus $migration_status */
+    $migration_status = \Drupal::service('ls_migration.migration_status');
+    if ($migration_status->isImporting()) {
+      return;
+    }
+
     // For backwards compatibility, modules that override the Media entity
     // class, are not required to implement the prepareSave() method.
     // @todo For Drupal 8.7, consider throwing a deprecation notice if the
